book/tut4_geophysics.ipynb

```diff
@@ -0,0 +1,1388 @@
+{
```

Comment:
TEST: Please change "gray" to "lightgray"

Reply:
Updated in commit https://github.com/GenericMappingTools/agu24workshop/pull/8/commits/04de4e7ca14b43555a9d9beaa677dfbd9e271ac6.